cypress/integration/Archiving-Deals.js

```diff
@@ -0,0 +1,303 @@
+      cy.get('[data-qa="contextual-targeting--exclude_Citadel"]').click();
+      cy.get('[data-qa="contextual-targeting--exclude_citadel_input"]').click().type('f55ffa62{enter}');
+      cy.get('[data-qa="audience-targeting--include_krux_input"]').click().type('qa-krux{enter}');
+      cy.get('[data-qa="audience-targeting--key_exclude_dropdown"]').click();
+      cy.get('[data-qa="audience-targeting--exclude_MAS"]').click();
+      cy.get('[data-qa="audience-targeting--exclude_mas_input"]').click().type('A{enter}');
+
+      // TECHNOGRAPHIC TARGETING
+      cy.get('[placeholder="+ Add Browser"]').click().type('firefox{enter}');
+      cy.get('[placeholder="+ Add Operating System"]').click().type('nintendo{enter}');
+      cy.get('[placeholder="+ Add ISP Target"]').click().type('digital{enter}');
+      cy.get('[placeholder="+ Add Carrier"]').click().type('at&t{enter}');
+      cy.get('[placeholder="+ Add Social"]').click().type('pin{enter}');
+      cy.get('[placeholder="+ Add Device"]').click();
+      cy.contains('Tablet', { timeout: 9000 }).click();
+
+      // GEO TARGETING
+      cy.get('[data-qa="geo-targeting--dropdown_type"]').click();
+      cy.get('[data-qa="geo-targeting--dropdown_type_EXCLUDE"]').click();
+      cy.get('[placeholder="Choose a location"]').focus().type('tokyo');
+      cy.contains('Region').click(); // selecting Region version option from dropdown
+      cy.get('[data-qa="targeting--save_targeting"]').click().wait(2400);
+
+      cy.get('[data-qa="deal-details--push_to_ad_server"]').click();
+      cy.wait(2000); // Adding this way to make sure data is pushed fully to external systems.
+
+      cy.get('p2:nth-child(2)').should('have', 'Tablet'); // Verifying Deal has tablet in devices section
+
+      // Grabbing targeting values from saved state of deal targeting section
+      cy.get('contextual-targeting section:nth-child(1) p2')
+        .invoke('text').then((text) => {
+          gsValue = text;
+        });
+      cy.get('contextual-targeting section:nth-child(2) p2')
+        .invoke('text').then((text) => {
+          citadelValue = text;
+        });
+      cy.get('audience-targeting div.layout--targeting-section section:nth-child(2) p2')
+        .invoke('text').then((text) => {
+          masValue = text;
+        });
+    });
+
+    it('Retrieve data for PFR Deal from SSP', () => {
+      const requestOptions = getRequest();
+      requestOptions.url = `https://deal-manager.dev.kargo.com/api/v1/deal/${dealID}/ssp`;
+
+      cy.request(requestOptions)
+        .then((resp) => {
+          assert.equal(resp.body.status, 1, 'response status ');
+          cy.log(dealID);
+          assert.equal(resp.body.name, `${dealName}_edited`, 'deal Name ');
+          assert.equal(resp.body.external_id, dealID, 'deal ID ');
+          assert.equal(resp.body.type, 'preferred_fixed_price', 'Deal Type ');
+        });
+    });
+
+    it('Retrieve data for PFR Deal from kraken', () => {
```

Comment:
Should be: Verify Deal info was pushed properly to Kraken.
or 
Verifying Deal Info on Kraken service